### PR TITLE
Moves vfs-modal scripts into tycho-biz.js + some codestyle

### PR DESCRIPTION
**Pull request by meg1502**

### CODE REVIEW

File: src/main/resources/default/taglib/t/fileField.html.pasta [hunk 1]
```diff
@@ -59,6 +59,7 @@
                 }
                 selectVFSFile({
                     path: value,
+                    allowUpload: true,
```
Comment (sabieber) [inline src/main/resources/default/taglib/t/fileField.html.pasta:62]: Was this change intended?

File: src/main/resources/default/assets/scripts/vfs-modals.js [hunk 2]
```diff
@@ -180,7 +180,7 @@ function selectVFSFile(config) {
                         })
                     }
                 } else {
-                    _uploadBox.textContent = '';
+                    _uploadBox.style.display = 'none';
```
Comment (sabieber) [inline src/main/resources/default/assets/scripts/vfs-modals.js:183]: Why not via bootstrap marker class?

**Review by sabieber — CHANGES_REQUESTED** [review]

**Review by andyHa — APPROVED** [review]

File: src/main/resources/default/taglib/t/fileField.html.pasta (hunk 1 above)
Comment (meg1502) [reply src/main/resources/default/taglib/t/fileField.html.pasta:62]: yes, this was previously set inside the script itself to always true. 
i would keep this here in case we might need to move it to a tag parameter or set it to false in another tag

**Review by sabieber — APPROVED** [review]

File: src/main/resources/default/taglib/t/fileField.html.pasta (hunk 1 above)
Comment (sabieber) [reply src/main/resources/default/taglib/t/fileField.html.pasta:62]: Okay, it just didnt match the commit message at all which hinted at a commit gone wrong

**Review by idlira — APPROVED** [review]
